HACKING.md

```diff
@@ -74,28 +73,23 @@ FAF should be available at ```http://localhost/faf/```
 ### Running container images
 Prerequisites:
 
-1. Docker, see for example [this
-guide](https://developer.fedoraproject.org/tools/docker/docker-installation.html)
+1. Podman, installed by default in Fedora
```

Comment:
That’s true for *default* default installations, but not for any of the spins (KDE, LXDE, etc.): https://pagure.io/fedora-comps/blob/master/f/comps-f33.xml.in. Looks for environments that include the `container-management` group.

Reply:
Do I have to specify that in what installations podman can be found, or is it enough to say that i can be installed by running dnf install podman _if needed_?

Reply:
That’s why I linked to https://podman.io/getting-started/installation before, so you could just add the link and forget about this.

Reply:
Oh, I see, sorry.